js/form.js

```diff
@@ -263,7 +263,7 @@
 
     var count = numberWithCommas(PetitionMap.current_petition.data.attributes.signature_count);
 
-    var sign_link = 'https://petition.parliament.uk/petitions/' + PetitionMap.current_petition.data.id + '/signatures/new';
+    var sign_link = 'https://petition.parliament.uk/petitions/' + PetitionMap.current_petition.data.id;
```

Comment:
Given the link no longer goes direct to the sign page, do we also want to change the text?  (I'm not sure that we do TBH, just seeing what other folk think)